app/v2/notifications/get_notifications.py

```diff
@@ -53,7 +53,7 @@ def get_pdf_for_notification(notification_id):
     except Exception:
         raise PDFNotReadyError()
 
-    return send_file(filename_or_fp=BytesIO(pdf_data), mimetype="application/pdf")
+    return send_file(path_or_file=BytesIO(pdf_data), mimetype="application/pdf")
```

Comment:
At the current moment, it is the only breaking change that I have noticed from upgrading Flask and Werkzeug to version 2.